NYPLAudiobookToolkit/UI/ScrubberView.swift

```diff
@@ -269,15 +270,13 @@ final class ScrubberView: UIView {
     }
     
     override func updateConstraints() {
-        super.updateConstraints()
-        UIView.beginAnimations("moveScrubberViewProgressBar", context: nil)
         self.barWidthConstraint?.constant = self.state.progressLocationFor(self.progressBarWidth)
         self.progressBar.backgroundColor = self.state.progressColor
         self.gripper.backgroundColor = self.trimColor
         self.labelWidthConstraints.forEach { (constraint) in
             constraint.constant = self.state.progress.labelWidth
         }
-        UIView.commitAnimations()
+        super.updateConstraints()
```

Comment:
in WWDC this year they advised against overriding `updateConstraints` for performance reasons. I think for updating color/modifying constants we can just move this to a new method `updateUI` thats called whenever we need to update

https://developer.apple.com/videos/play/wwdc2018/220/

Reply:
There may or may not be good reasons to not override `updateConstraints()` but this particular change was just moving the superclass call to the bottom of the method, where it is explained in the method docs:
```
Important
Call [super updateConstraints] as the final step in your implementation.
```
It's the opposite of things like `viewDidLoad()` because the subviews should be first to update their layouts.